profiles/seccomp/seccomp_default.go

```diff
@@ -1608,27 +1536,332 @@ var DefaultProfile = &types.Seccomp{
 			Action: types.ActAllow,
 			Args:   []*types.Arg{},
 		},
-		// i386 specific syscalls
-		{
-			Name:   "modify_ldt",
-			Action: types.ActAllow,
-			Args:   []*types.Arg{},
-		},
-		// arm specific syscalls
-		{
-			Name:   "breakpoint",
-			Action: types.ActAllow,
-			Args:   []*types.Arg{},
-		},
-		{
-			Name:   "cacheflush",
-			Action: types.ActAllow,
-			Args:   []*types.Arg{},
-		},
-		{
-			Name:   "set_tls",
-			Action: types.ActAllow,
-			Args:   []*types.Arg{},
-		},
-	},
+	}
+
+	var arch string
+	var native, err = libseccomp.GetNativeArch()
+	if err == nil {
+		arch = native.String()
+	}
+	switch arch {
+	case "arm", "arm64":
+		syscalls = append(syscalls, []*types.Syscall{
+			{
+				Name:   "breakpoint",
+				Action: types.ActAllow,
+				Args:   []*types.Arg{},
+			},
+			{
+				Name:   "cacheflush",
+				Action: types.ActAllow,
+				Args:   []*types.Arg{},
+			},
+			{
+				Name:   "set_tls",
+				Action: types.ActAllow,
+				Args:   []*types.Arg{},
+			},
+		}...)
+	case "amd64", "x32":
+		syscalls = append(syscalls, []*types.Syscall{
+			{
+				Name:   "arch_prctl",
+				Action: types.ActAllow,
+				Args:   []*types.Arg{},
+			},
+		}...)
+		fallthrough
+	case "x86":
+		syscalls = append(syscalls, []*types.Syscall{
+			{
+				Name:   "modify_ldt",
+				Action: types.ActAllow,
+				Args:   []*types.Arg{},
+			},
+		}...)
+	}
+
+	capSysAdmin := false
+
+	var cap string
+	for _, cap = range rs.Process.Capabilities {
+		switch cap {
+		case "CAP_CHOWN":
```

Comment:
just glancing over this, so this may be a silly note; are these the values as passed through the command-line (I.e, `--cap-add`), because I think we pass those without the `CAP_` prefix; see https://docs.docker.com/engine/reference/run/#runtime-privilege-and-linux-capabilities